src/main/java/rs117/hd/resourcepacks/Constants.java

```diff
@@ -0,0 +1,55 @@
+package rs117.hd.resourcepacks;
+
+import com.google.gson.Gson;
+import net.runelite.client.RuneLite;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.commons.text.WordUtils;
+import rs117.hd.resourcepacks.data.Manifest;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+
+public class Constants {
+    public static OkHttpClient CLIENT = new OkHttpClient();
+    public static Gson GSON = new Gson();
```

Comment:
I think you're required to inject the client's OkHttpClient and Gson instances rather than creating your own.

ref: https://github.com/runelite/plugin-hub/commit/78630b8